reactive-banana/src/Reactive/Banana/Combinators.hs

```diff
@@ -277,12 +277,13 @@ observeE = E . Prim.observeE . Prim.mapE unM . unE
 -- | Dynamically switch between 'Event'.
 -- Semantically,
 --
--- > switchE ee = \time0 -> concat [trim t1 t2 e | (t1,t2,e) <- intervals ee, time0 <= t1]
+-- > switchE e0 ee = \time0 -> [(t, a) | (t,a) <- e0, t < t1] ++ concat [trim t1 t2 e | (t1,t2,e) <- intervals ee, time0 <= t1]
```

Comment:
This might also be

```suggestion
-- > switchE e0 ee = \time0 -> [(t, a) | (t,a) <- e0, t <= t1] ++ concat [trim t1 t2 e | (t1,t2,e) <- intervals ee, time0 <= t1]
```

We should confirm this before merging.